website/src/components/visualisation/Resources/LineGraph.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { Brush, ResponsiveContainer, LineChart, Line, YAxis, XAxis, CartesianGrid, Tooltip, Legend, Surface, Symbols, ReferenceLine } from '../../../../node_modules/recharts';
+import outputJSON from '../../../output/output.json';
+import _ from "../../../../node_modules/lodash";
+
+
+class LineRechartComponent extends React.Component<{}, any> {
```

Comment:
haven't seen class components in some time... but this actually looks like a neat use case for class, I'm fine with this. We usually use hooks as per other files.

(I started react with class components, and still have a soft spot for them!)